crates/tx5-connection/src/config.rs

```diff
@@ -35,3 +36,39 @@ pub struct HubConfig {
     #[cfg(test)]
     pub test_fail_webrtc: bool,
 }
+
+/// Configuration for a group of ICE servers.
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase")]
+pub struct IceServers {
+    /// The ICE server URLs to use for discovering external candidates.
+    pub urls: Vec<String>,
+}
+
+/// WebRTC config.
+///
+/// This configuration will be mapped the specific configuration used by
+/// the selected backend.
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase")]
+pub struct WebRtcConfig {
+    /// A list of ICE servers configurations.
+    pub ice_servers: Vec<IceServers>,
+}
```

Comment:
```suggestion
pub struct WebRtcConfig {
    /// A list of ICE servers configurations.
    pub ice_servers: Vec<IceServers>,
    
    #[serde(default)]
    pub ice_transport_policy: TransportPolicy,
}

#[derive(Debug, Clone, Serialize, Deserialize, Default)]
#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
#[serde(rename_all = "camelCase")]
pub enum TransportPolicy {
    #[default]
    All,
    Relay,
}
```

I think we may also need the transport policy to be available for certain configurations.

Reply:
Added